src/attributes/attributes.js

```diff
@@ -0,0 +1,49 @@
+(function () {
+	htmx.defineExtension('attributes', {
+		handleSwap: function (swapStyle, target, fragment) {
+			var source = fragment.firstElementChild || fragment;
+
+			if (swapStyle === 'attributes') {
+				var isOobSwap = source.attributes['hx-swap-oob'];
+				if (isOobSwap) {
+					var oobSelector = source.attributes['id'].value;
+					var actualTarget = htmx.find('#' + oobSelector);
+					if (actualTarget) {
+						this.swapAttributes(actualTarget, source);
+					}
+				} else {
+					this.swapAttributes(target, source);
+				}
+				return true;
+			}
+			return false;
+		},
+
+		swapAttributes: function (target, source) {
```

Comment:
Hi stuckennedy I think this function should not be returned as a base element of the extension definition as this is designed really only for the 6 or so overridable functions and while it will probably not break anything it can add confusion.  This function could be defined with inline scope just like the forEach function inside it does or moved out of the scope of the extension definition to the top maybe.